chemicaltoolbox/autodock_vina/docking/docking.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="docking" name="Docking" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.1.2">autodockvina</requirement>
```

Comment:
``` yml
name: package_autodockvina_1_1_2
owner: leobiscassi
description: Contains a tool dependency definition that downloads and extracts version 1.1.2 of autodock vina.
homepage_url: http://vina.scripps.edu/index.html
long_description: |
  AutoDock Vina is an open-source program for doing molecular docking. 
  It was designed and implemented by Dr. Oleg Trott in the Molecular Graphics Lab at The Scripps Research Institute.
type: tool_dependency_definition
categories:
  - Tool Dependency Packages
```
``` yml
name: package_mgltools_1_5_6
owner: leobiscassi
description: Contains a tool dependency definition that downloads and extracts version 1.5.6 of MGL Tools.
homepage_url: http://mgltools.scripps.edu/
long_description: |
  MGLTools is a software developed at the Molecular Graphics Laboratory (MGL)
  of The Scripps Research Institute for visualization and analysis of molecular structures. 
type: tool_dependency_definition
categories:
  - Tool Dependency Packages
```
I made this files, how can you see above, but I not included in this commit. In this case, where the tool dependencies file contains two packages, how I'll create the shed file? It would be how bellow?

``` yml
name: package_mgltools_1_5_6
owner: leobiscassi
description: Contains a tool dependency definition that downloads and extracts version 1.5.6 of MGL Tools.
homepage_url: http://mgltools.scripps.edu/
long_description: |
  MGLTools is a software developed at the Molecular Graphics Laboratory (MGL)
  of The Scripps Research Institute for visualization and analysis of molecular structures. 
type: tool_dependency_definition
categories:
  - Tool Dependency Packages

name: package_autodockvina_1_1_2
owner: leobiscassi
description: Contains a tool dependency definition that downloads and extracts version 1.1.2 of autodock vina.
homepage_url: http://vina.scripps.edu/index.html
long_description: |
  AutoDock Vina is an open-source program for doing molecular docking. 
  It was designed and implemented by Dr. Oleg Trott in the Molecular Graphics Lab at The Scripps Research Institute.
type: tool_dependency_definition
categories:
  - Tool Dependency Packages
```